### PR TITLE
Support collection pagination

### PR DESCRIPTION
**Pull request by nglx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,14 +137,17 @@ export.status
...
 
-To get all the panels:
+To paginate all the panels:
 
 ```ruby
-QualtricsAPI.panels.fetch
+panels = QualtricsAPI.panels.fetch
+# => #<QualtricsAPI::PanelCollection:0x007f8769aae2c0 ....>
+
+panels = panels.next # get next page if not panels.last?
```
Comment (pallymore) [inline README.md]: can we rename it to something like `next_page` ? 
`next` sounds like it's fetching a single record.

File: README.md [hunk 2]
```diff
@@ -35,14 +35,20 @@ end
...
 
 ```ruby
-QualtricsAPI.surveys.fetch
+surveys = QualtricsAPI.surveys.fetch
 # => #<QualtricsAPI::SurveyCollection:0x007fcb72cce350 ....>
+
+surveys = surveys.next # to fetch next page if not on the last page
+# => #<QualtricsAPI::SurveyCollection:0x007fcb72cce350 ....>
+
+surveys.last?
```
Comment (pallymore) [inline README.md]: same with `last?` maybe we can just say `next_page?` or `last_page?`

File: README.md (hunk 1 above)
Comment (nglx) [inline README.md]: ok

File: README.md (hunk 2 above)
Comment (nglx) [inline README.md]: yeah will change to `next_page?`

**Pull request merged**